CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.12.2
+
```

Comment:
Can you repeat the whole `IMPORTANT: Docker 1.12 ships with an updated systemd  ... ` section for this release? Not everyone reads older release notes, so perhaps better "safe" than "sorry" for those that do not check older versions :smile: